packages/wonder-blocks-core/src/hooks/__tests__/use-is-mounted.test.tsx

```diff
@@ -24,24 +25,22 @@ describe("useIsMounted", () => {
 
         it("should return false on initial render", () => {
             // Arrange
-            jest.spyOn(console, "error").mockImplementation(() => {
-                // We need to capture the initial render only, without mounting
-                // so we can't use Testing Library. Instead, we're going to
-                // use `renderToString` from `react-dom/server`, but that is
-                // going to log an error because we use `useLayoutEffect`.
-                // So, we suppress that error for our testing purposes.
-                /* no-op */
-            });
+            // We cannot use renderHook on useIsMounted directly here and then
+            // look at the result.all[0] to see the first render as the result
+            // is a function that accesses the underlying ref, and that ref will
+            // have changed to `true` even if it was `false` in the initial
+            // render. So, we use a component.
```

Comment:
**note:** @kevinbarabash, I realised today why `result.all` would not work for checking the initial render. This new test follows the spirit of your original suggestion though :)

Reply:
I think another approach here would likely be to have a hook in the test that wraps the `useIsMounted` hook and returns the result of `isMounted` as its return value, then render that hook. That way the `result.all` approach would work, I think.